### ISSUE TITLE
can we calculate gradients at special point while custom optimizer?

### ISSUE THREAD

**Issue by bojone** [keras-team/keras#10565]
I want to custom my own optimizer.

if loss = L(paras), K.gradients(loss, paras) return L'(paras).

is there any way to return L'(2 * paras) , for example? 

the original Nesterov optimizer also need the similar operation.

**Comment by li2109** [comment]
I am facing the same question here. Did you find a way to solve this？

**Comment by VIAIMAKE** [comment]
I have encountered the same question...
